README.md

```diff
@@ -73,27 +87,17 @@ The simulator has its own configuration,
 you can refer to the [documentation](./simulator/docs/simulator-server-config.md).
 
 See the following docs to know more about simulator:
+- [existing-cluster-importing.md](simulator/docs/import-cluster-resources.md): describes how you can import resources in your cluster to the simulator so that you can simulate scheduling based on your cluster's situation.
```

Comment:
```suggestion
- [import-cluster-resources.md](./simulator/docs/import-cluster-resources.md): describes how you can import resources in your cluster to the simulator so that you can simulate scheduling based on your cluster's situation.
```